var/spack/repos/builtin/packages/r-rmpi/package.py

```diff
@@ -31,9 +31,9 @@ class RRmpi(RPackage):
 
     homepage = "http://www.stats.uwo.ca/faculty/yu/Rmpi"
     url      = "https://cran.r-project.org/src/contrib/Rmpi_0.6-6.tar.gz"
+    list_url = "https://cran.rstudio.com/web/packages/Rmpi/index.html"
 
-    version('0.6-6', '59ae8ce62ff0ff99342d53942c745779')
-
+    version('0.6-6', 'a6fa2ff5e1cd513334b4e9e9e7a2286f')
```

Comment:
Did the checksum changed?

Reply:
Please confirm!

Reply:
Yes. The older one does not match when installing.